docs/proposals/scheduling/crd-scheduling-improvements/crd-scheduling-improvements.md

```diff
@@ -0,0 +1,138 @@
+---
+title: CRD Component Scheduler Estimation
+authors:
+  - "@mszacillo"
+  - "@Dyex719"
+reviewers:
+- "@RainbowMango"
+- "@XiShanYongYe-Chang"
+- "@zhzhuang-zju"
+approvers:
+- "@RainbowMango"
+
+create-date: 2024-06-17
+---
+# CRD Component Scheduler Estimation
+
+## Summary
+
+Users may want to use Karmada for resource-aware scheduling of Custom Resources (CRDs). This can be done
+if the CRD is comprised of a single podTemplate, which Karmada can already parse if the user defines
+the ReplicaRequirements with this in mind. Resource-aware scheduling becomes more difficult however,
+if the CRD is comprised of multiple podTemplates or pods of differing resource requirements.
+
+In the case of FlinkDeployments, there is only one podTemplate per CRD. However this podTemplate contains information
```

Comment:
```suggestion
In the case of [FlinkDeployments](https://raw.githubusercontent.com/apache/flink-kubernetes-operator/release-1.9/examples/basic.yaml), there is only one podTemplate per CRD. However this podTemplate contains information
```
Maybe we can leave a link to FlinkDeployments for people who aren't familiar with it.